docs/getting-started/windows-standalone.md

```diff
@@ -0,0 +1,55 @@
+# Windows Standalone
+
+A Windows MSI package (64-bit only) is available to download at
+[https://github.com/signalfx/splunk-otel-collector/releases
+](https://github.com/signalfx/splunk-otel-collector/releases).
+
+## Getting Started
+
+### GUI
+
+Double-click on the downloaded package and follow the wizard.
+
+### CLI
+
+TODO
```

Comment:
would it be better to take this out until there's info?

Reply:
I think we should can remove this since the "Powershell" section below describes how to install it from the command line.

Reply:
I was going to specific MSI silent install options, but will remove for now